### ISSUE TITLE
Shouldn't it work with non-ASCII characters.

### ISSUE THREAD

**Issue by kurt-krueckeberg** [Einenlum/linguee-api#3]
This doesn't work

```php
<?php
use Einenlum\LingueeApi\Factory;
include "vendor/autoload.php";

$linguee = Factory::create();
$response = $linguee->translate('Zucker', 'ger', 'eng'); // Works
 print_r ($response->toArray());

$response = $linguee->translate('vernachlässigen', 'ger', 'eng'); // <-- Fails. Passing German characters like ä, ö, ü, and ß fails
 print_r ($response->toArray());
```

It fails with
```bash
PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\Component\DomCrawler\Crawler::getText() in /home/kurt/temp/vendor/einenlum/linguee-api/src/Response/Transformer/Word/Translations.php:74
```

**Comment by einenlum** [comment]
Thanks for your feedback.
I'm working on it. Hope to fix it soon.

**Comment by einenlum** [comment]
Just fixed it in [v0.1.1](https://github.com/Einenlum/linguee-api/releases/tag/v0.1.1).

Should work now. Thanks for your feedback, and please open another issue if you find another problem :wave:.

**Comment by kurt-krueckeberg** [comment]
Yes, it seems to be working now. Thanks a lot. This is really something helpful to me.
